README.md

```diff
@@ -1 +1,179 @@
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+> This is my recent Portfolio showcasing some of my recent work.
+
+**[Portfolio]** is a website portfolio...
+
+## 🛠 Built With <a name="built-with"></a>
+
+**[HTM]**
+**[CSS]**
+**[Javascript]**
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Responsive Design]**
+- **[Good accesibility]**
+- **[Higher SEO]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://prg-04.github.io/Portfolio_/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+- First you need to have an IDE installed in your PC. You also need to have a   github account in order to clone the code to your local repository
+- Login to your github account and search for 'prg-04/Portfolio_'
+- Click the green 'code' button and copy the HTTPS url.
+- Open the your preferred IDE and open the terminal window. Write the         following commands in the terminal 'git clone' followed by the url link you copied and press Enter on your keyboard. Git will start cloning the repository into your selected folder.
+- After cloning the repository change the directory using 'cd Portfolio_' to access the files.
+
+### Prerequisites
+
+In order to run this project you need:
+
+    * Visual Studio Code,Atom or your preferred IDE
+    * MAC, Linux, Windows Vista or newer -GitHub account
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open the your preferred IDE and open the terminal window. Write the following commands in the terminal 'git clone' followed by the url link you copied and press Enter on your keyboard. Git will start cloning the repository into your selected folder.
+- After cloning the repository change the directory using 'cd Portfolio_' to access the files.
+
+### Install
+
+Install this project with:
+
+- In your terminal window type: git clone https://github.com/prg-04/Portfolio_.git
+
+### Usage
+
+To run the project, execute the following command:
+
+- code Portfolio_ .
+
+### Run tests
+
+To run tests, run the following command:
+
+- npx stylelint "\*_/_.{css,scss}"
+- npx eslint .
```

Comment:
- We are not working on testing in this portfolio project. Why don't we replace the`To run tests, run the following command:` to `To run linter checks, run the following command:` because we are not testing the actual code but instead, checking to see if we are following code best practices, spacing, indentations e.t.c. 🌸